dask/array/percentile.py

```diff
@@ -38,6 +38,8 @@ def percentile(a, q, interpolation='linear'):
     if not a.ndim == 1:
         raise NotImplementedError(
             "Percentiles only implemented for 1-d arrays")
+    if isinstance(q, (int, float)):
```

Comment:
Can I recommend the use of `numbers.Number` here instead?  That will support NumPy scalars or other number-like objects.  Otherwise this looks great to me!

Reply:
Good point! I'll use `numbers.Number` instead